src/shared-components/dropdown/mobileDropdown.tsx

```diff
@@ -4,9 +4,21 @@ import PropTypes from 'prop-types';
 import ChevronIcon from '../../svgs/icons/chevron-icon.svg';
 import { DropdownContainer, dropdownInputStyle, IconContainer } from './style';
 
+import { OptionType } from './index';
+
+type MobileDropdownProps = {
```

Comment:
It's not part of the diff, so I can't comment directly on it, but [L77 (old diff)/L95 (new diff)](https://github.com/curology/radiance-ui/pull/261/files#diff-d73843a82a787f2c9d2271ef4eea7cd7R95) is spreading `{...this.props}` into this component. So we're also passing this component two unused props: `onChange` and `optionsContainerMaxHeight`.

My recommendation would be to pass down `options`, `textAlign`, and `value` explicitly, outside of the spread, on the aforementioned L77/L95.